include/flatbuffers/base.h

```diff
@@ -23,13 +26,6 @@
 #include <cstdlib>
 #include <cstring>
 
-#if defined(FLATBUFFERS_MEMORY_LEAK_TRACKING) && \
```

Comment:
I think originally this was put here to avoid clashes with the include files above.. how are we sure this compiles, since we don't build with `FLATBUFFERS_MEMORY_LEAK_TRACKING` on?

Reply:
Checked and fixed in #5105